packages/utils/src/worldwide.ts

```diff
@@ -44,34 +68,23 @@ export interface InternalGlobal {
    */
   _sentryDebugIds?: Record<string, string>;
   __SENTRY__: {
-    hub: any;
-    logger: any;
-    extensions?: {
-      /** Extension methods for the hub, which are bound to the current Hub instance */
-      // eslint-disable-next-line @typescript-eslint/ban-types
-      [key: string]: Function;
-    };
-    globalScope: Scope | undefined;
-    defaultCurrentScope: Scope | undefined;
-    defaultIsolationScope: Scope | undefined;
-    globalMetricsAggregators: WeakMap<Client, MetricsAggregator> | undefined;
-    /** Overwrites TextEncoder used in `@sentry/utils`, need for `react-native@0.73` and older */
-    encodePolyfill?: (input: string) => Uint8Array;
-    /** Overwrites TextDecoder used in `@sentry/utils`, need for `react-native@0.73` and older */
-    decodePolyfill?: (input: Uint8Array) => string;
-  };
+    [key: VersionString]: BackwardsCompatibleSentryCarrier;
+    version: VersionString;
```

Comment:
I guess this should be:

```suggestion
    version?: VersionString;
```

Reply:
Good catch, thx! fixed in [b74ee3f](https://github.com/getsentry/sentry-javascript/pull/12206/commits/b74ee3f54ab00eb432f970ca2af9405c8d50ee05)